mappings/net/minecraft/client/gui/screen/multiplayer/SocialInteractionsPlayerListEntry.mapping

```diff
@@ -8,21 +8,30 @@ CLASS net/minecraft/unmapped/C_wolpvava net/minecraft/client/gui/screen/multipla
 	FIELD f_edbalobk offline Z
 	FIELD f_hadcnich showButton Lnet/minecraft/unmapped/C_buwziidm;
 	FIELD f_irhuazit client Lnet/minecraft/unmapped/C_ayfeobid;
+	FIELD f_ivvohkov HIDE_MESSAGES_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_jlsahfax hasSentMessages Z
 	FIELD f_jwjfvksx hideTooltip Ljava/util/List;
+	FIELD f_jxwxlwvx REPORT_BUTTON Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_ktatsldl PADDING I
+	FIELD f_marlhjad wrappedReportText Ljava/util/List;
 	FIELD f_mqmjljjj showTooltip Ljava/util/List;
 	FIELD f_mwbfkocl BLOCKED_OFFLINE_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_okxewylp REPORT_DISABLED_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_olbgyiom HIDDEN_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_oqfmxozt OFFLINE_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_ovmwamcr REPORT_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_pezawfyv BLOCKED_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_pfudqoys MAX_TOOLTIP_WIDTH I
+	FIELD f_pypfkory reportable Z
```

Comment:
```suggestion
	FIELD f_pypfkory isReportable Z
```
Is just a bit clearer in my opinion

Reply:
imo `is` is meant for method names

Reply:
There's no explicit definition of this in the contributing guidelines, but I believe that using `is` is clearer for the user